src/js/components/TabContent.tsx

```diff
@@ -24,13 +26,16 @@ export default function TabContent() {
   const dispatch = useDispatch()
   const space = useSelector(Current.getSpace)
   const conn = useSelector(Current.getConnection)
-  const connStatus = useSelector(ConnectionStatuses.get(conn.id))
+  const id = get(conn, ["id"], "")
+  const connStatus = useSelector(ConnectionStatuses.get(id))
```

Comment:
This is just an idea for the future, not a suggestion for this PR. We could make Current.getConnection also make the join with the ConnectionStatuses and add that to the object that it returns. Then we could just write conn.status.